doc/htmldoc/templates/index.html

```diff
@@ -37,9 +37,9 @@
 <nav id="topnav" role="navigation">
  <div class="menu-toggle">Menu</div>
     <ul class="srt-menu" id="menu-main-navigation">
- <li><a href="#intro">Get started</a></li>
-	<li><a href="#topics">Learn NEST</a></li>
-	<li><a href="#backmatter">Ecosystem</a></li>
+       <li><a href="https://nest-initiative.org/">The NEST Initiative</a></li>
```

Comment:
```suggestion
       <li><a href="https://nest-simulator.org/">The NEST Homepage</a></li>
```
I think it makes more sense to have a link to the nest-simulator.org

Reply:
These changes should also be applied to the `templates/layout.html` so the buttons are consistent across all pages.